client/disputes/style.scss

```diff
@@ -1,3 +1,11 @@
+// dispute list table css override
+.disputes-list {
```

Comment:
I wonder if this CSS class is better prefixed with `wcpay-` like the other classes below it.